upgrade/1.5.2/CSM-Only.md

```diff
@@ -0,0 +1,146 @@
+# CSM Only Upgrade
+
+This page provides guidance for systems with exclusively CSM installed that are performing an upgrade from a CSM v1.5.X release to the CSM v1.5.2 release.
+
+The [v1.5.2 upgrade page](../1.5.2/README.md) will refer to this page
+during [Update NCN images](../1.5.2/README.md#update-ncn-images).
+
+**If other products are installed on the system, return to [Update NCN images](../1.5.2/README.md#update-ncn-images) and
+choose option 1.**
+
+## Requirements
+
+* `CSM_RELEASE` is set in the shell environment on `ncn-m001`. See [Preparation](README.md#preparation) for details on how it should be set.
+
+## Steps
+
+1. (`ncn-m001#`) Generate a new [CFS](../../glossary.md#configuration-framework-service-cfs) configuration for the management nodes.
+
+   This script creates a new CFS configuration that includes the CSM version in its name and applies it to the
+   management nodes. This leaves the management node components in CFS disabled. They will be automatically enabled when
+   they are rebooted at a later stage in the upgrade.
+
+   ```bash
+   /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
+       --no-enable --config-name "management-${CSM_RELEASE}"
+   ```
+
+   Successful output should end with the following line:
+
+   ```text
+   All components updated successfully.
+   ```
+
+1. (`ncn-m001#`) Set the [IMS](../../glossary.md#image-management-service-ims) IDs for each Management node sub-role.
+
+    * Kubernetes ID
+
+      ```bash
+      KUBERNETES_IMAGE_ID="$(kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' \
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-kubernetes")) | $o[.].id')"
+      export KUBERNETES_IMAGE_ID
```

Comment:
Do we need to export this? Looks like we're just using it as a command argument later.

```suggestion
```